slang-gfx.h

```diff
@@ -1445,6 +1445,13 @@ class IDevice: public ISlangUnknown
                 return handles;
             }
 
+            // This creates and returns an ExistingDeviceHandles object as values is zeroed by default.
```

Comment:
This does not need to be defined in the public header file since it is only used by the implementation.